### PR TITLE
MR-59

### PR DESCRIPTION
**Pull request by baldhat**
Added event throwing for Runtime Exceptions that are not part of the domain. For example Connection Problems with other Services now lead to an event being thrown.

### CODE REVIEW

File: src/main/kotlin/com/msd/application/dto/GameMapPlanetDto.kt [hunk 1]
```diff
@@ -9,6 +9,7 @@ class GameMapPlanetDto(
     val id: UUID,
     @JsonProperty("movement_difficulty")
     val movementDifficulty: Int,
+    @JsonProperty("planet_type")
```
Comment (DennisWae) [inline src/main/kotlin/com/msd/application/dto/GameMapPlanetDto.kt:12]: Why ist this here? Don't we use camelCase?

File: src/main/kotlin/com/msd/command/application/CommandApplicationService.kt [hunk 2]
```diff
@@ -127,7 +127,7 @@ class CommandApplicationService {
                 RepairItemType.valueOf(args[1].uppercase()),
                 UUID.fromString(args[2])
             )
-            else -> throw RuntimeException("Internal Error")
+            else -> throw RuntimeException("Unknown command verb $verb")
```
Comment (DennisWae) [inline src/main/kotlin/com/msd/command/application/CommandApplicationService.kt:130]: I think it would be nicer if we used a custom exception here

File: src/test/kotlin/com/msd/application/GameMapServiceTest.kt [hunk 3]
```diff
@@ -21,20 +22,16 @@ class GameMapServiceTest(
...
-    companion object {
-        val mockGameServiceWebClient = MockWebServer()
+    val mockGameServiceWebClient = MockWebServer()
 
-        @BeforeAll
-        @JvmStatic
-        internal fun setUp() {
-            mockGameServiceWebClient.start(port = 8081)
-        }
+    @BeforeEach
```
Comment (DennisWae) [inline src/test/kotlin/com/msd/application/GameMapServiceTest.kt:27]: Why did you change this to a `BeforeEach`? Doesn't it now take longer to run those tests?

File: src/test/kotlin/com/msd/command/application/CommandControllerTest.kt [hunk 4]
```diff
@@ -77,21 +77,7 @@ class CommandControllerTest(
     private lateinit var robot8: Robot
     private lateinit var robots: List<Robot>
 
-    companion object {
-        val mockGameServiceWebClient = MockWebServer()
-
-        @BeforeAll
-        @JvmStatic
-        internal fun setUp() {
-            mockGameServiceWebClient.start(port = 8081)
```
Comment (DennisWae) [inline src/test/kotlin/com/msd/command/application/CommandControllerTest.kt:86]: Same here

File: src/main/kotlin/com/msd/application/dto/GameMapPlanetDto.kt (hunk 1 above)
Comment (baldhat) [reply src/main/kotlin/com/msd/application/dto/GameMapPlanetDto.kt:12]: This is what we receive from the Map Service, and they use snake

File: src/test/kotlin/com/msd/application/GameMapServiceTest.kt (hunk 3 above)
Comment (baldhat) [reply src/test/kotlin/com/msd/application/GameMapServiceTest.kt:27]: This somehow fixed the mockServer stopping to work after a few test methods

File: src/main/kotlin/com/msd/command/application/CommandApplicationService.kt (hunk 2 above)
Comment (baldhat) [reply src/main/kotlin/com/msd/command/application/CommandApplicationService.kt:130]: This actually cannot be thrown, but I'll change it anyways
